StripeConnect/StripeConnect/Source/Internal/Webview/ConnectWebViewController.swift

```diff
@@ -131,7 +131,7 @@ private extension ConnectWebViewController {
     func openInAppSafari(url: URL) {
         let safariVC = SFSafariViewController(url: url)
         safariVC.dismissButtonStyle = .done
-        safariVC.modalPresentationStyle = .popover
+        safariVC.modalPresentationStyle = .pageSheet
```

Comment:
This also caused a crash on iPad because we weren't setting a sourceView. I meant to use pageSheet here from the beginning, it's just that popover and pageSheet do the same thing on iPhone.